include/fi_util.h

```diff
@@ -147,20 +149,21 @@ struct util_prov {
 /*
  * Fabric
  */
+struct util_ns; 
 struct util_fabric_info {
 	const char 			*name;
 	const struct fi_provider 	*prov;
 };
 
 struct util_fabric {
-	struct fid_fabric	fabric_fid;
-	struct dlist_entry	list_entry;
-	fastlock_t		lock;
-	ofi_atomic32_t		ref;
-	const char		*name;
-	const struct fi_provider *prov;
+	struct fid_fabric		fabric_fid;
+	struct dlist_entry		list_entry;
+	fastlock_t			lock;
+	ofi_atomic32_t			ref;
+	const char			*name;
+	const struct fi_provider	*prov;
 
-	struct dlist_entry	domain_list;
+	struct dlist_entry		domain_list;
```

Comment:
The changes of "struct util_fabric" now becomes purely cosmetic and can be excluded from the commit.